shell/app-shell/app-shell.js

```diff
@@ -200,7 +201,6 @@ const template = ArcsUtils.html`
     <arc-handle arc='{{arc}}' data='{{arcsHandleData}}' options='{{arcsHandleOptions}}' on-change='_onArcsHandleChange'></arc-handle>
     <arc-handle arc='{{arc}}' data='{{identityHandleData}}' options='{{identityHandleOptions}}' on-change='_onIdentityHandleChange'></arc-handle>
     <arc-handle arc='{{arc}}' data='{{identitiesHandleData}}' options='{{identitiesHandleOptions}}' on-change='_onIdentitiesHandleChange'></arc-handle>
-    <arc-handle arc='{{arc}}' data='{{friendsAvatarData}}' options='{{friendsAvatarHandleOptions}}'></arc-handle>
```

Comment:
Can we also delete lines 308 - 316 in `app-shell.js` or are you planning to put this back in some form later? Note `vr-app-shell.js` also has this `arc-handle`.

Reply:
There are no commented out lines in `app-shell.js` that are there by accident. But I'm getting the message that people are not used to this, so I will change my behavior and be more aggressive about deleting such code.

Reply:
Those lines aren't commented out -- they build the data that it looked like this handle was the only place using.

Reply:
Sorry, my bad, was looking at the wrong thing. You are totally right, will remove.